src/utils/portfolioCalculations/calculateInvestedAmount.ts

```diff
@@ -1,6 +1,25 @@
+import { CurrentPortfolioItem } from '@/types/currentPortfolio';
 import { FX_RATE } from './const/FX_RATE';
 import { calculationParams } from '@/types/calculations';
 
+export const calculateCurrentInvestedAmount = (
+  currentPortfolio: CurrentPortfolioItem[],
+) => {
+  const totalInvestedAmountWithFees = currentPortfolio.reduce((sum, tx) => {
+    return sum + (tx.value.total + (tx.totalFees || 0)) * FX_RATE;
+  }, 0);
+  const totalInvestedAmountNoFees = currentPortfolio.reduce((sum, tx) => {
+    return sum + tx.value.total * FX_RATE;
+  }, 0);
+
+  const investedAmount = {
+    withFees: totalInvestedAmountWithFees,
+    noFees: totalInvestedAmountNoFees,
+  };
+
+  return investedAmount;
+};
+
```

Comment:
Just small cosmetic suggestions. Number of characters could be reduced how to write this function and in the same time maintain it's readability. Less characters = less things to read = easier to understand. Of course, this is not this simple, finding the right balance will come with experience. 
```ts
export const calculateCurrentInvestedAmount = (
  currentPortfolio: CurrentPortfolioItem[],
) => {
  const totalInvestedAmountWithFees = currentPortfolio.reduce(
    (sum, tx) => sum + (tx.value.total + (tx.totalFees || 0)) * FX_RATE,
    0,
  );
  const totalInvestedAmountNoFees = currentPortfolio.reduce(
    (sum, tx) => sum + tx.value.total * FX_RATE,
    0,
  );

  return {
    withFees: totalInvestedAmountWithFees,
    noFees: totalInvestedAmountNoFees,
  };
};
```